src/Containers/AddTodo/Action.js

```diff
@@ -0,0 +1,13 @@
+let nextTodoId = 0;
```

Comment:
It's seems wrong

Reply:
Yes, it is wrong.
I submitted a PR with a fix that passes a props nextTodoId from AddTodo as parameter to this Action. Action must be a pure function, that is can't have a assignments, API calls or side effects